doc/specs/#147 - Support different release channels [released, beta, alpha].md

```diff
@@ -0,0 +1,259 @@
+Downloading https://dl.google.com/dl/chrome/install/googlechromestandaloneenterprise64.msi
+  ██████████████████████████████  78.8 MB / 78.8 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+
+PS> winget install Google.Chrome --channel dev
+Found Google Chrome [Google.Chrome (Dev)] Version 97.0.4692.20
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://dl.google.com/tag/s/dl/chrome/install/dev/googlechromedevstandaloneenterprise64.msi
+  ██████████████████████████████  78.4 MB / 78.4 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+### Listing installed packages
+```raw
+PS> winget list
+Name                                    Id                                       Version            Available    Channel Source
+-------------------------------------------------------------------------------------------------------------------------------
+Microsoft Edge                          Microsoft.Edge                           92.0.902.67        95.0.1020.53         winget
+Microsoft Edge Update                   Microsoft Edge Update                    1.3.147.37
+App Installer                           Microsoft.DesktopAppInstaller_8wekyb3d8… 1.16.12653.0                    Stable 
+Microsoft Edge                          Microsoft.MicrosoftEdge.Stable_8wekyb3d… 92.0.902.67                     Stable
+Google Chrome                           Google.Chrome                            96.0.4664.45       97.0.4692.20 Dev     winget
+```
+
+## Capabilities
+
+Many packages use different structures for release channels. The Windows Package Manager should accommodate as many of these as is practicable.
+
+### Accessibility
+
+The Windows Package Manager has been built in such a way that screen readers will still provide audible output as the command is executed keeping the user informed of progress, warnings, and errors. This should have no direct impact on accessibility.
+
+### Security
+
+There should be no security impact directly, although we must remember that different sources may not guarantee the safety of packages. The Windows Package Manager community repository performs static and dynamic analysis, and in some cases additional manual validation before accepting a package.
+
+### Reliability
+
+The Windows Package Manager community repository has many packages that may be affected by release channels. The ease of keeping packages updated must be a consideration when designing other tools for creating and updating manifests.
+
+### Compatibility
+
+The current implementation for different release channels is to have separate package identifiers for each package. Packages which have channel-specific identifiers currently existing in the community repository /should/ be updated to use this feature, but will not be required to.
```

Comment:
If they do not support side-by-side installation, they really should be moved to this pattern.

If they do support it, I think that is a whole other design spec to deal with them better.

Reply:
I think the point of channels is to have them be combined to a single identifier where possible. The packages which do support side by side are the best to have in the Example.Package / Example.Package.Dev format. It is when they do not support side by side that they should have the channel specified for the installer to ensure the proper channel is used for updates and installation